examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -30,6 +30,7 @@ model:
   sample_rate: 16000
   log_prediction: true # enables logging sample predictions in the output during training
   ctc_reduction: 'mean_batch'
+  skip_nan_grad: false
```

Comment:
Any reason this is false? Also, I believe Megatron already has code to detect and skip batches with nan grads (and also correct the step counter for the scheduler)- Look at https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py#L342-L378 

Can we utilize similar technique at the ASRModel level so that automatically all models will benefit from this?

Reply:
Also why is the default false? It should be good to always skip nan grads no?

Reply:
It makes training slightly slower and it did not help this model.